src/components/Button/index.view.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Not entirely sure why this is here, but please enable this rule again by deleting this line.

Reply:
I copied this template from the `PostcardStack`, so I'd recommend we check that too.

Reply:
We need it in the `PostcardStack` since we use `require` within that file to import the SVGs.